docs/NewOperators.md

```diff
@@ -0,0 +1,24 @@
+## How to implement a new operator in Glow
+
+*Note: It's best to avoid implementing new Glow IR nodes and instructions. If support for a new operator is needed, it's best to first check to see if this operator can be implemented using an existing glow IR node or nodes. For example FCTransposed can be implemented using a regular FC node. If this is the case then the operator can be implemented by instantiating existing glow IR nodes in the loader or by simply creating a new node creation method in `Graph/Graph.cpp` which would create the sequence of operators that implements the new operator. Only in the event that this is not possible should new Glow IR nodes/instructions be created.*
+
+> Example PR: https://github.com/pytorch/glow/pull/2334
+
+### Implementation
+#### High level IR
+* Create a new Glow high level IR node in `ClassGen/NodeGen.cpp`. Run `ninja all` to generate the node.
```

Comment:
My understanding is, we can run "ninja all" for release/debug build, and check the AutoGenNodes.h/cpp file to make sure the node is added?  If so, maybe we can explain more here :)